pkg/resmgr/nri.go

```diff
@@ -492,35 +513,137 @@ func (p *nriPlugin) updateResources(u *api.ContainerUpdate, c cache.Container) {
 	}
 }
 
-func (p *nriPlugin) dump(header string, args ...interface{}) {
-	for {
-		var (
-			prefix string
-			msg    []byte
-			ok     bool
-			err    error
-		)
+const (
+	request  = "=>"
+	response = "<="
+)
+
+func (p *nriPlugin) dump(dir, event string, args ...interface{}) {
+	// request dumping
+	if dir == request {
+		switch {
+		case strings.HasSuffix(event, "Sandbox"):
+			if len(args) != 1 {
+				p.Error("invalid dump(%s, %s, %d args...), expecting <pod>",
+					dir, event, len(args))
+				return
+			}
+
+			pod := args[0].(*api.PodSandbox)
+			p.Info("%s %s %s/%s", dir, event, pod.GetNamespace(), pod.GetName())
+
+			// for RunPodSandbox, dump details if configured so
+			if event == "RunPodSandbox" && p.DebugEnabled() {
```

Comment:
Sorry in advance if I'm missing something, but there is no defaulting or a separate case here that would cover PostConfig-Update event. Are we missing the entry for that here?

Reply:
There is absolutely no need to apologize. We really want to encourage review comments/questions and not discourage them. Therefore, IMO there are no bad/stupid review questions. If in doubt, it is better to ask/comment than to stay silent. An unasked question never helps and it might leave something bad unfixed that was overlooked/missed by the PR author... especially if that author was me. ;-)